src/Factory/Configuration.php

```diff
@@ -0,0 +1,39 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Laminas\I18n\PhoneNumber\Factory;
+
+use Laminas\I18n\CountryCode;
+use Laminas\I18n\PhoneNumber\ConfigProvider;
+use Psr\Container\ContainerInterface;
+
+/**
+ * @psalm-import-type PackageConfig from ConfigProvider
+ * @psalm-internal \Laminas\I18n\PhoneNumber
+ */
+abstract class Configuration // phpcs:ignore
```

Comment:
From what I can see, you're consuming the static methods from this class, but never extending it. Would it make more sense to make this class final instead?

Reply:
with a private constructor if its not supposed to be called

Reply:
Yes - it's effectively `Utils`!! - updated to final with private constructor. Already marked as internal.